system-tests/fixtures/api/ComplexApi/package.json

```diff
@@ -11,7 +11,7 @@
     "message": {
       "moduleName": "ComplexApi",
       "namespace": "com.ComplexApi.ern",
-      "apiSchemaPath": "/Users/blemair/Code/electrode-native/system-tests/fixtures/api/complexapi-schema.json",
+      "apiSchemaPath": "/Users/f0b00n7/github/electrode-io/electrode-native/system-tests/fixtures/api/complexapi-schema.json",
```

Comment:
pls, check if this is intentional. IMHO, none is correct, is CI workspace set up as /Users/blemair/Code/?

Reply:
These files are auto-generated. It's not "intentional" ;)